src/dosbox.cpp

```diff
@@ -916,17 +916,26 @@ const char *filter_on_or_off[] = {"on", "off", 0};
 	                  "  on:   Filter the output (default).\n"
 	                  "  off:  Don't filter the output.");
 
-	// Disney Audio emulation
-	secprop->AddInitFunction(&DISNEY_Init,true);//done
-
-	Pbool = secprop->Add_bool("disney", when_idle, true);
-	Pbool->Set_help("Enable Disney Sound Source emulation. (Covox Voice Master and Speech Thing compatible).");
-
-	Pstring = secprop->Add_string("disney_filter", when_idle, "on");
-	Pstring->Set_help("Filter for the Disney audio output:\n"
+	// LPT DAC device emulation
+	secprop->AddInitFunction(&LPT_DAC_Init, true);
+	const char *lpt_dac_types[] = {"none", "disney", "covox", "ston1", "off", 0};
+	pstring = secprop->Add_string("lpt_dac", when_idle, lpt_dac_types[0]);
+	pstring->Set_help("Type of DAC(s) to plug into the parallel port:\n"
+	                  "  disney:   Disney Sound Source, in mono at 7 kHz.\n"
+	                  "  covox:    Covox Speech Thing, in mono at the mixer's rate.\n"
+	                  "  ston1:    Stereo-on-1 DAC, in stereo up to 30 kHz.\n"
+	                  "  none/off: No DACs are provided (default).\n");
+	pstring->Set_values(lpt_dac_types);
+
+	pstring = secprop->Add_string("lpt_dac_filter", when_idle, "on");
+	pstring->Set_help("Filter for the LPT DAC audio device(s):\n"
```

Comment:
No plural, and I'd suggest this text for consistency with the other filter descriptions:

`Filter for the LPT DAC output`

Reply:
Perfect, thanks!